app/authorisation.js

```diff
@@ -1,6 +1,20 @@
 const config = require('../config')
 
-var isAuthenticated = function (req) {
+var isUserAuthenticated = function (req) {
+  var result = false
+  if (isAuthenticationEnabled()) {
+    if (!req.isAuthenticated() || !req.user) {
+      result = false
+    } else {
+      result = true
+    }
+  } else {
+    result = true
+  }
+  return result
```

Comment:
From what I can see, everywhere we use `isUserAuthenticated`, we redirect them to login. This method could do the redirection and we would remove the duplication, which would make it easier to change if required. The method name should be renamed in that case to something like: `assertUserAuthenticated`.